pkg/storkctl/clusterpair.go

```diff
@@ -27,10 +28,11 @@ import (
 )
 
 const (
-	clusterPairSubcommand = "clusterpair"
-	cmdPathKey            = "cmd-path"
-	gcloudPath            = "./google-cloud-sdk/bin/gcloud"
-	gcloudBinaryName      = "gcloud"
+	clusterPairSubcommand  = "clusterpair"
+	cmdPathKey             = "cmd-path"
+	gcloudPath             = "./google-cloud-sdk/bin/gcloud"
+	gcloudBinaryName       = "gcloud"
+	skipResourceAnnotation = "stork.libopenstorage.org/skip-resource"
```

Comment:
nit: Can you define this as a global constant somewhere and change all the references in cli and controllers?

Reply:
Not implementing this for now as it this constant is also used outside of packages in volumes also. Will implement in another PR.